platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationImplTest.java

```diff
@@ -77,15 +81,33 @@ public void testAppGetters() throws Exception {
                         .size());
     }
 
+    @Ignore
```

Comment:
Can you add a reason for why this is ignored or a ticket when it can be reenabled?